lib/core/tests/html_renderer.rs

```diff
@@ -81,16 +81,16 @@ mod tests {
                 object.set_position(Vector3::new(x, y, z));
 
                 // Creates a gradient color based on the axis.
-                let r = (x * 25.5) as u8;
-                let g = (y * 25.5) as u8;
-                let b = (z * 25.5) as u8;
+                let r = (x * 50.0) as u8;
+                let g = (y * 50.0) as u8;
+                let b = (z * 50.0) as u8;
                 let color = format!("rgba({}, {}, {}, {})", r, g, b, 1.0);
 
                 object.dom.set_property_or_panic("background-color", color);
                 scene.add(object);
             }
         }
-        assert_eq!(scene.len(), 30, "We should have 30 HTMLObjects");
+        assert_eq!(scene.len(), 15, "We should have 30 HTMLObjects");
```

Comment:
Change the error message. Or even better, make `let expected_objects = 15` and use it in both places